Week-6/exercise-6.5/diff.js

```diff
@@ -0,0 +1,23 @@
+let arr = [5, 10, 3, 2, 50, 80];
+let target = 78;
+
+const targetDiff = () => {
+  for (let i = 0; i < arr.length - 1; i++) {
+    for (j = i + 1; j < arr.length; j++) {
+      if (target === mDiff(arr[i], arr[j])) {
+        return 1;
+      }
+    }
+  }
+  return 0;
```

Comment:
Time complexicity is O(n^2) , using a hash map this can be acheived in linear time with extra space.